youths/schema.py

```diff
@@ -213,7 +213,9 @@ def mutate_and_get_payload(cls, root, info, **input):
         )
 
         profile = Profile.objects.get(user=info.context.user)
-        youth_profile, created = YouthProfile.objects.get_or_create(profile=profile)
+        youth_profile, created = YouthProfile.objects.get_or_create(
```

Comment:
so, you will assign the attributes twice? a few lines below:
```python
for field, value in input_data.items():
    setattr(youth_profile, field, value)
```

Reply:
🤦‍♂ missed this. I still need to use the defaults youth profile does not exist. What about checking `created`?

Reply:
yeah, I think that's what you would need to do